unitTests/controllers/QueryControllerTest.ts

```diff
@@ -331,6 +349,50 @@ export function QueryControllerTest() {
       });
     });
 
+    describe('when calling fetchQueryExecutionPlan', () => {
+      it('should call createQueryBuilder', () => {
+        spyOn(test.cmp, 'createQueryBuilder').and.callThrough();
+        test.cmp.fetchQueryExecutionPlan();
```

Comment:
By not awaiting this call, is there a risk this call might still be running during the execution of the next test?

Reply:
Hmm great point!